packages/react-server-cli/src/compileClient.js

```diff
@@ -181,7 +181,6 @@ function packageCodeForBrowser(entrypoints, outputDir, outputUrl, hot, minify, l
 				{
 					test: /\.css$/,
 					loader: extractTextLoader,
```

Comment:
@mtrabelsi Trailing commas in object literals are supported in node.

We actually [_require_ them in multi-line object literals](https://github.com/redfin/react-server/blob/master/.eslintrc#L27) like this specifically so we don't need to clutter our diffs with modification of the previous line when we add/remove the last item in an object literal. ;)

Thanks for the bug report and for looking at he patch!